controller/api/Deliveries.php

```diff
@@ -192,17 +192,8 @@ public function publish(ServerRequest $request)
                 throw new common_exception_NotImplemented('Only POST method is accepted to publish deliveries');
             }
             $requestData = $request->getParsedBody();
-            if (!array_key_exists(self::REST_DELIVERY_URI, $requestData)) {
-                throw new common_exception_RestApi(__('Missing required parameter: `%s`', self::REST_DELIVERY_URI), 400);
-            }
-            if (!array_key_exists(self::REST_REMOTE_ENVIRONMENTS, $requestData)) {
-                throw new common_exception_RestApi(__('Missing required parameter: `%s`', self::REST_REMOTE_ENVIRONMENTS), 400);
-            }
-            $deliveryUri = $requestData[self::REST_DELIVERY_URI];
-            $remoteEnvironmentUris = $requestData[self::REST_REMOTE_ENVIRONMENTS];
-            if (!is_array($remoteEnvironmentUris)) {
-                $remoteEnvironmentUris = [$remoteEnvironmentUris];
-            }
+            $deliveryUri = $this->getDeliveryParameter($requestData);
```

Comment:
getDeliveryIdParameter?

Reply:
Renamed to `getDeliveryUriParameter`

Reply:
about this URI: I'm OK with that, because all project uses it, but it is more like an ID, example: when we move it to KV storage it becomes kv_de_uri, so for delivery it could be kv_d_uri